dataprofiler/data_readers/data_utils.py

```diff
@@ -301,7 +300,46 @@ def detect_file_encoding(file_path, buffer_size=1024, max_lines=20):
     encoding = detector.result["encoding"]
 
     # Typical file representation is utf-8 instead of ascii, treat as such.
-    if not encoding or encoding == 'ascii':
+    if not encoding or encoding == 'ascii' or encoding == 'Windows-1254':
```

Comment:
Why is ascii not okay?

Reply:
I would suggest using lower here as well and using an in statement to clean up the if statement:
```python
if not encoding or encoding.lower() in ['ascii', 'windows-1254']:
    ...
```

Reply:
Added in new commit